docs/platforms/android/profiling/index.mdx

```diff
@@ -28,7 +28,7 @@ By default, some transactions will be created automatically for common operation
 
 <Note>
 
-Android profiling is available starting in SDK version `6.16.0`.
+Android profiling is available starting in SDK version `6.16.0` and requires API level 22 or higher.
```

Comment:
Does it implies that the user should set minimum API 22?
Is there another way of saying it?
Something like `and works on API 22 and higher`, but better 😄

Reply:
reworded it a little bit :)